Security Consideration: add a section related to CSP (#4975)
DevExpress/devextreme-documentation: concepts/Common/Security Considerations/40 Content Security Policy/05 Common CSP Directives.md

> There are a set of different directives that you can implement in a CSP, to fit the security needs of your website or application.

Reviewer: ```suggestion
You can implement a set of different directives in CSP to fit the security needs of your website or application.
```

Do not use 'there is/there are' in your texts.

> To get started, create a list of policies or directives to define which resources your site will allow or restrict. In the table below, we provided a list of common CSP directives and source values for you to mix and match.

Reviewer: ```suggestion
To get started, create a list of policies or directives to define which resources your site will allow or restrict. See the list of common CSP directives and source values in the table below:
```

Do not ever use 'provide'.

<table border="1"  cellspacing="0" cellpadding="4">
    <tbody>
        <tr>
            <th style="width: 20%;"><b>Directive</b></th>
            <th><b>Example</b></th>
            <th><b>Description</b></th>
        </tr>
        <tr>
            <td>default-src</td>

>             <td>default-src 'self' cdn.example.com;</td>

Reviewer: Wrap examples in `<code>` tag
